test/helpers/web.test.js

```diff
@@ -1,106 +1,102 @@
-const nock = require("nock");
+const nock = require('nock')
 
-const webHelper = require("../../src/helpers/web");
+const webHelper = require('../../src/helpers/web')
 
-describe("Web Helpers", function() {
-  let inputs;
-  let uploadURL;
-  let token;
-  let uploadFile;
-  let version;
-  let query;
-  beforeEach(function() {
-    inputs = { args: {}, envs: {} };
```

Comment:
my node isn't great, I'm guessing this doesn't need to be declared here anymore?

Reply:
Which line? (keeping mind this is a linting PR)